modules/core/test/v2/unit/wallet.ts

```diff
@@ -1599,7 +1599,7 @@ describe('V2 Wallet:', function () {
 
       it('should build an enable token transaction', async function () {
         const recipients = [];
-        const token = 'tokenName';
+        const tokenName = 'tokenName';
```

Comment:
avoid using value as same variable name, this can hide potential error. You can use anything different like: 'tokenName-123'

Reply:
changed